lib/features/reps_core/reps_core.views_default.inc

```diff
@@ -182,24 +150,50 @@ function reps_core_views_default_views() {
     'reps_core_sb_right_free_content' => 'reps_core_sb_right_free_content',
     'reps_core_sb_right_image_link' => 'reps_core_sb_right_image_link',
   );
-  /* Filter criterion: Block: Position (field_reps_rightcol_position) */
-  $handler->display->display_options['filters']['field_reps_rightcol_position_value']['id'] = 'field_reps_rightcol_position_value';
-  $handler->display->display_options['filters']['field_reps_rightcol_position_value']['table'] = 'field_data_field_reps_rightcol_position';
-  $handler->display->display_options['filters']['field_reps_rightcol_position_value']['field'] = 'field_reps_rightcol_position_value';
-  $handler->display->display_options['filters']['field_reps_rightcol_position_value']['value'] = array(
-    0 => '0',
+  $handler->display->display_options['filters']['type']['group'] = 1;
+  /* Filter criterion: Block: Type */
+  $handler->display->display_options['filters']['type_1']['id'] = 'type_1';
+  $handler->display->display_options['filters']['type_1']['table'] = 'bean';
+  $handler->display->display_options['filters']['type_1']['field'] = 'type';
+  $handler->display->display_options['filters']['type_1']['value'] = array(
+    'webtools' => 'webtools',
+  );
+  $handler->display->display_options['filters']['type_1']['group'] = 2;
+  /* Filter criterion: Block: Display (field_reps_display_region) */
+  $handler->display->display_options['filters']['field_reps_display_region_value']['id'] = 'field_reps_display_region_value';
+  $handler->display->display_options['filters']['field_reps_display_region_value']['table'] = 'field_data_field_reps_display_region';
+  $handler->display->display_options['filters']['field_reps_display_region_value']['field'] = 'field_reps_display_region_value';
+  $handler->display->display_options['filters']['field_reps_display_region_value']['value'] = array(
+    'right_sidebar' => 'right_sidebar',
   );
+  $handler->display->display_options['filters']['field_reps_display_region_value']['group'] = 2;
+  $handler->display->display_options['block_description'] = 'Right column - reps beans';
 
-  /* Display: Right column top - All pages */
-  $handler = $view->new_display('block', 'Right column top - All pages', 'block_3');
+  /* Display: Right column  -  reps bean */
+  $handler = $view->new_display('block', 'Right column  -  reps bean', 'block_2');
```

Comment:
Isn't it a duplicate of "block_reps_bean" ?

Reply:
yes right I made a view test ans I forgot to delete it.